cyclonedx-lib/src/temurin/sbom/TemurinGenSBOM.java

```diff
@@ -27,10 +27,21 @@
 import org.cyclonedx.model.Property;
 import org.cyclonedx.model.Tool;
 import org.cyclonedx.parsers.JsonParser;
+import org.cyclonedx.exception.ParseException;
+import org.webpki.json.JSONAsymKeySigner;
+import org.webpki.json.JSONObjectWriter;
+import org.webpki.json.JSONOutputFormats;
+import org.webpki.json.JSONParser;
+import org.webpki.util.PEMDecoder;
+import com.google.gson.JsonParser;
```

Comment:
com.google.gson.JsonParser is invalid, remove this import as JsonParser comes from "import org.cyclonedx.parsers.JsonParser" already further up